src/lib/coerce.js

```diff
@@ -173,7 +174,22 @@ exports.valObjectMeta = {
         requiredOpts: [],
         otherOpts: ['dflt'],
         coerceFunction: function(v, propOut, dflt) {
-            propOut.set(scales.get(v, dflt));
+            propOut.set(colorscales.get(v, dflt));
+        }
+    },
+    opacityscale: {
```

Comment:
We don't a new `valType` just for `opacityscale`. In the `volume` attribute, you can set `valType` to `any` and coerce "manually" in the `volume/defaults.js`.